sklearn/svm/tests/test_svm.py

```diff
@@ -202,6 +199,41 @@ def test_linearsvr():
     assert np.abs(score1 - score2) < 0.1
 
 
+def test_linearsvr_fit_sampleweight():
+    # check correct result when sample_weight is 1
+    # check that SVR(kernel='linear') and LinearSVC() give
+    # comparable results
+    diabetes = datasets.load_diabetes()
+    n_samples = len(diabetes.target)
+    unit_weight = np.ones(n_samples)
+    lsvr = svm.LinearSVR(C=1e3).fit(diabetes.data, diabetes.target,
+                                    sample_weight=unit_weight)
```

Comment:
It may also be worth testing that the method accepts a list (rather than array).